x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_size_property.js

```diff
@@ -121,25 +121,28 @@ export class DynamicSizeProperty extends DynamicStyleProperty {
   }
 
   getMbSizeExpression() {
-    if (this._isSizeDynamicConfigComplete(this._options)) {
-      return this._getMbDataDrivenSize({
-        targetName: this.getComputedFieldName(),
-        minSize: this._options.minSize,
-        maxSize: this._options.maxSize,
-      });
+    if (!this._isSizeDynamicConfigComplete(this._options)) {
+      return null;
     }
-    return null;
+
+    return this._getMbDataDrivenSize({
+      targetName: this.getComputedFieldName(),
+      minSize: this._options.minSize,
+      maxSize: this._options.maxSize,
+    });
   }
 
```

Comment:
You also need to update `syncIconSizeWithMb` method to use range instead of 0 and 1.